### ISSUE TITLE
I don't get how the scanner handles specials / oad

### ISSUE THREAD

**Issue by whinette** [ZeroQI/Absolute-Series-Scanner#389]
Hello,
I've recently updated both the scanner and hama after ~2 years as I am recently rewatching/adding anime.

I don't get how the scanner now handles the specials of seasons. I will take https://anidb.net/anime/14526/relation/graph as an example.
I've named `S1` the specials of seasons 2 part 1 and part 2
https://anidb.net/anime/14767 : Tensei Shitara Slime Datta Ken (2021) [anidb-14767]/S1 [EP-24.9]-Digression~Hinata Sakaguchi [78063161]
https://anidb.net/anime/15456 : Tensei Shitara Slime Datta Ken (2021 Dai 2 Bu) [anidb-15456]/S1 [SP]-Veldora's Journal 2 [DBBAEFF8]

Even though the folder has the correct [anidb-] tag and that the episode in each folders are correctly mapped to anidb, both these specials are matched to the special of first season https://anidb.net/episode/212772

I also have issue with this season that is only composed with oav: https://anidb.net/anime/14526
The oav are named as usual under a folder with the correct [anidb-] (exemple for the first one: Tensei Shitara Slime Datta Ken (2019) [anidb-14526]/E01 -The Tragedy of M [2E129247].mkv)
The season is correctly matched (correct name, correct description from anidb) BUT the oav have the metadata of the episode from the season 1 like the specials from the other seasons ...

I've cleaned the cache of the plugin.
I've removed the whole folder of this anime, scanned, cleared trash, re added the folder in path, scan, for the same result

Logs:
[Tensei Shitara Slime Datta Ken.agent-search.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/7875889/Tensei.Shitara.Slime.Datta.Ken.agent-search.log)
[Tensei Shitara Slime Datta Ken.agent-update.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/7875890/Tensei.Shitara.Slime.Datta.Ken.agent-update.log)

Capture from the OAV season:
![image](https://user-images.githubusercontent.com/6152660/149637812-3b4aacf5-f0c5-47d8-a4b8-0b23cd215074.png)

![image](https://user-images.githubusercontent.com/6152660/149637831-4887a5fd-2ffb-4ac2-bcad-f725cd473db1.png)

**Comment by ZeroQI** [comment]
You have raised a scanner issue so you have to demonstrate the season or episode number gets mapped wrong, but no scanner logs present to asserting that. 

You seem to say the season and episode number in Plex is correct but showing the wrong metadata, that's an agent issue then...

**Comment by whinette** [comment]
You're right, I wan't looking to the correct logs. They look ok, we need to move the ticket. Sorry about that.

[Tensei Shitara Slime Datta Ken.scanner.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/7875919/Tensei.Shitara.Slime.Datta.Ken.scanner.log)
[Tensei Shitara Slime Datta Ken.filelist.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/7875920/Tensei.Shitara.Slime.Datta.Ken.filelist.log)
.

**Comment by ZeroQI** [comment]
If you set anidbid the episodes must be season 1.

You forced anidbid with season 2 numbering when anidb never have seasons beyond 1. The following has wrong season numbering:
- Tensei Shitara Slime Datta Ken\Tensei Shitara Slime Datta Ken (2021 Dai 2 Bu) [anidb-15456]

Because there is a season 2 numbering, it uses tvdb numbering and find tvdbid through the mapping file, the special ends up being tvdb s00e01.

**Comment by whinette** [comment]
Ok, this explain why the episodes were matched even though they were absolute numbering for the season 2 part 2.
I've renamed everything and it fixes the issues for the two specials of the s2 wrongly named with s02. Thank you.

anidb-14526 gave me more problems. Whatever I did the titles were still pulled from TVDB. On a hunch, I renamed them to S1-S5 and voilà, correctly matched.

Thanks. Now I will recheck my whole library to find inconsistencies. ;)
